meshtastic/__init__.py

```diff
@@ -238,4 +245,7 @@ def _receiveInfoUpdate(iface, asDict):
     portnums_pb2.PortNum.STORE_FORWARD_APP: KnownProtocol("storeforward", storeforward_pb2.StoreAndForward),
     portnums_pb2.PortNum.NEIGHBORINFO_APP: KnownProtocol("neighborinfo", mesh_pb2.NeighborInfo),
     portnums_pb2.PortNum.MAP_REPORT_APP: KnownProtocol("mapreport", mqtt_pb2.MapReport),
+    portnums_pb2.PortNum.ADMIN_APP: KnownProtocol(
```

Comment:
I think this is just replacing it, but we should probably just add the handler to the existing entry for `ADMIN_APP` up on line 228

Reply:
Missed that it was already in there!